public/images/image-cat-set.svg

```diff
@@ -1,134 +1,235 @@
-        <path fill="#BFEC80" d="M35.7661751,12.3796493 C35.7661751,17.5839606 32.5279099,22.0239602 27.9633899,23.7700953 C9.23757969,22.0688435 16.4113336,2.16005462 26.0981527,0.441540005 C31.6186778,1.57441315 35.7661751,6.49000366 35.7661751,12.3796493"/>
-        <path fill="#AAE655" d="M18.3629042,35.1349061 C17.0274695,35.6480437 15.578818,35.9268383 14.0637225,35.9268383 C7.38348931,35.9268383 1.96656415,30.4713529 1.96656415,23.74446 C1.96656415,17.0141146 7.38348931,11.5581976 14.0637225,11.5581976 C14.8968939,11.5581976 15.7117057,11.6440802 16.4972298,11.8063507 C6.8104108,13.5248654 -0.362906009,33.4332227 18.3629042,35.1349061"/>
-        <path fill="#BFEC80" d="M26.1655582,23.7444169 C26.1655582,28.9487282 22.9272931,33.3887278 18.362773,35.1348629 C-0.363037149,33.4331796 6.81071679,13.5248222 16.4975358,11.8063076 C22.018061,12.9391807 26.1655582,17.8547712 26.1655582,23.7444169"/>
-        <path fill="#759684" d="M26.7119718,22.7314772 C26.1318992,22.1587829 25.1920679,22.1587829 24.6119953,22.7314772 L24.1774873,23.1608901 L24.1774873,20.8360186 C24.1774873,20.026392 23.5126113,19.3699729 22.692991,19.3699729 C21.8729336,19.3699729 21.2080576,20.026392 21.2080576,20.8360186 L21.2080576,26.09255 L20.7731124,25.6631371 C20.1930398,25.0904428 19.2536457,25.0904428 18.6735731,25.6631371 C18.0935005,26.2358315 18.0935005,27.163277 18.6735731,27.7359713 L21.2080576,30.2382184 L21.2080576,44.2892979 L24.1774873,44.2892979 L24.1774873,27.3061269 L26.7119718,24.8043114 C27.2916072,24.2316171 27.2916072,23.3041716 26.7119718,22.7314772"/>
-        <g opacity=".5" transform="translate(0 32.368)">
-          <mask id="image-cat-set-l" fill="#fff">
-            <use xlink:href="#image-cat-set-k"/>
-          </mask>
-          <path fill="#F3FCFF" d="M44.4294086,0.226358843 L44.4294086,1.08993186 C44.4294086,2.45240062 43.308168,3.55937952 41.9277089,3.55937952 L2.5006943,3.55937952 C1.10843263,3.55937952 -0.000131139247,2.45240062 -0.000131139247,1.08993186 L-0.000131139247,0.226358843 L2.37829767,0.226358843 L2.37829767,0.50601667 C2.37829767,1.15078333 3.36359054,1.67384704 4.59586233,1.67384704 L39.5873105,1.67384704 C40.8073426,1.67384704 41.8053123,1.15078333 41.8053123,0.50601667 L41.8053123,0.226358843 L44.4294086,0.226358843 Z" mask="url(#image-cat-set-l)"/>
-        </g>
-        <path fill="#847264" d="M18.6523722,45.9332371 L26.5862967,45.9332371 C26.5862967,45.1223157 25.9205464,44.4646019 25.0987405,44.4646019 C24.9431219,44.4646019 24.7953717,44.4948119 24.6537413,44.5388321 C24.2061193,43.9031284 23.4629969,43.4853679 22.6193344,43.4853679 C21.7761091,43.4853679 21.0329867,43.9031284 20.5849276,44.5388321 C20.4437343,44.4948119 20.2964212,44.4646019 20.1399284,44.4646019 C19.3185596,44.4646019 18.6523722,45.1223157 18.6523722,45.9332371"/>
-      </g>
-    </g>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same here -- if I open this in Sketch, delete the marks, then copy out the SVG code, I get a layout that matches what's currently on master.